explore-analyze/_snippets/variable-control-procedure.md

```diff
@@ -0,0 +1,22 @@
+1. While editing your {{esql}} query, the autocomplete menu suggests adding a control when relevant or when typing `?` in the query. Select **Create control**.
+
+   ![ESQL query prompting to add a control](/explore-analyze/images/esql-visualization-control-suggestion.png " =40%")
+
+2. A menu opens to let you configure the control. This is where you can specify:
+
+    * The type of the control. 
+      * For controls with **Static values**, enter available controls manually or select them from the dropdown list. 
+      * For controls with **Values from a query**, write an {{esql}} query to populate the list of options.
+    * The name of the control. This name is used to reference the control in {{esql}} queries. 
+      * Start the name with `?` if you want the options to be simple static values.
+      * Start the name with `??` if you want the options to be fields or functions. {applies_to}`stack: ga 9.1`
+    * The values users can select for this control. You can add multiple values from suggested fields, or type in custom values. If you selected **Values from a query**, you must instead write an {{esql}} query at this step.
+    * The label of the control. This is the label displayed in **Discover** or in the dashboard.
+    * The width of the control.
```

Comment:
just another option, these bullets could cut out some words by saying "The control's ...." Instead of "The ... of the control"